examples/examples.json

```diff
@@ -113,6 +113,21 @@
                 }
             ]
         },
+        {
+            "title": "Sprites",
+            "samples": [
+                {
+                    "title": "Single Sprite",
+                    "desc": "Show a sprite in the place of points",
+                    "file": "sprites/sprite.html"
+                },
+                {
+                    "title": "Multiple Sprites",
+                    "desc": "Show different sprites based on a category",
+                    "file": "sprites/sprite-ramp.html"
```

Comment:
I would rename it `multiple-sprites.html` so its the same as the title and to another example: `multiple-layers.html`

Reply:
:+1: